plugins/modules/ec2_vpc_nacl.py

```diff
@@ -300,306 +358,208 @@ def process_rule_entry(entry, Egress):
     else:
         if entry[6] or entry[7]:
             params["PortRange"] = {"From": entry[6], "To": entry[7]}
-    return params
-
-
-def restore_default_associations(assoc_ids, default_nacl_id, client, module):
-    if assoc_ids:
-        params = dict()
-        params["NetworkAclId"] = default_nacl_id[0]
-        for assoc_id in assoc_ids:
-            params["AssociationId"] = assoc_id
-            restore_default_acl_association(params, client, module)
-        return True
-
 
-def construct_acl_entries(nacl, client, module):
-    for entry in module.params.get("ingress"):
-        params = process_rule_entry(entry, Egress=False)
-        params["NetworkAclId"] = nacl["NetworkAcl"]["NetworkAclId"]
-        create_network_acl_entry(params, client, module)
-    for rule in module.params.get("egress"):
-        params = process_rule_entry(rule, Egress=True)
-        params["NetworkAclId"] = nacl["NetworkAcl"]["NetworkAclId"]
-        create_network_acl_entry(params, client, module)
+    return params
 
 
-# Module invocations
-def setup_network_acl(client, module):
+def construct_acl_entries(client, module: AnsibleAWSModule, nacl_id: str) -> bool:
```

Comment:
What about splitting this one too (one for egress and one for ingress)?

Reply:
done